modules/ROOT/pages/intro-configuration.adoc

```diff
@@ -3,23 +3,33 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-In Mule 3, Spring property placeholders are often used to configure apps dynamically for the environment in which they are deployed. Mule 4 contains a built-in mechanism for this that allows you to set default values and avoid the need to learn Spring.
+In Mule 3, Spring property placeholders are often used to configure apps dynamically for the environment in which they are deployed. Mule 4 contains a built-in mechanism for this that allows you to set default values and avoid the need to learn Spring. +
```

Comment:
I understand the impulse to begin with Mule 3 and then tell how Mule 4 is better, but maybe we should consider getting directly to why Mule 4 is better? Maybe something like this:
```suggestion
Mule 4 contains a built-in mechanism for configuring apps dynamically for the environment in which they are deployed, enabling you to set default values in a YAML file rather than learning how to use Spring property placeholders. +
```
OR
```suggestion
Although Mule 3 required you to know how to use Spring property placeholders to configure apps dynamically for the environment in which they are deployed, Mule 4 contains a built-in mechanism that enables you to set default values in a YAML file. +
```